draft-js-video-plugin/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "draft-js-video-plugin",
+  "version": "0.0.4",
+  "description": "video Plugin for DraftJS",
+  "author": {
+    "name": "Anchen li"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/draft-js-plugins/draft-js-plugins.git"
+  },
+  "main": "lib/index.js",
+  "keywords": [
+    "editor",
+    "video",
+    "draft",
+    "react",
+    "ux",
+    "components",
+    "widget",
+    "react-component"
+  ],
+  "scripts": {
+    "test": "node_modules/.bin/mocha --compilers js:babel-core/register --require testHelper.js **/__test__/*.js",
+    "clean": "./node_modules/.bin/rimraf lib",
+    "build": "npm run clean && npm run build:js",
+    "build:js": "WEBPACK_CONFIG=$(pwd)/webpack.config.js BABEL_DISABLE_CACHE=1 BABEL_ENV=production NODE_ENV=production ./node_modules/.bin/babel --out-dir='lib' --ignore='__test__/*' src",
+    "prepublish": "npm run build"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "draft-js": ">=0.8.1",
+    "immutable": ">=3.8.1",
+    "react": ">=15.2.1",
+    "react-dom": ">=15.2.1"
+  },
+  "devDependencies": {
+    "alex": "^3.1.0",
+    "animateplus": "^1.4.0",
+    "autoprefixer": "^6.4.1",
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.14.0",
+    "babel-eslint": "^6.1.2",
+    "babel-jest": "^15.0.0",
+    "babel-loader": "^6.2.5",
+    "babel-plugin-react-transform": "^2.0.2",
+    "babel-plugin-webpack-loaders": "^0.7.1",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-react-hmre": "^1.1.1",
+    "babel-preset-stage-0": "^6.5.0",
+    "chai": "^3.5.0",
+    "chai-enzyme": "^0.5.1",
+    "cheerio": "^0.20.0",
+    "css-loader": "^0.25.0",
+    "css-modules-require-hook": "^4.0.2",
+    "cz-conventional-changelog": "^1.1.6",
+    "dirty-chai": "^1.2.2",
+    "enzyme": "^2.4.1",
+    "eslint": "^3.5.0",
+    "eslint-config-airbnb": "11.0.0",
+    "eslint-plugin-jsx-a11y": "^2.2.1",
+    "eslint-plugin-mocha": "^4.5.1",
+    "eslint-plugin-react": "^6.2.0",
+    "estraverse-fb": "^1.3.1",
+    "express": "^4.14.0",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "flow-bin": "^0.32.0",
+    "jest-cli": "^15.1.1",
+    "jsdom": "^9.5.0",
+    "lint-staged": "^3.0.1",
+    "mocha": "^3.0.2",
+    "multer": "^1.2.0",
+    "postcss-loader": "^0.13.0",
+    "pre-commit": "^1.1.3",
+    "prismjs": "^1.5.1",
+    "react": "^15.2.1",
+    "react-addons-pure-render-mixin": "^15.2.1",
+    "react-addons-test-utils": "^15.2.1",
+    "react-dom": "^15.2.1",
+    "react-router": "^2.8.0",
+    "react-static-webpack-plugin": "^1.2.0",
+    "react-transform-catch-errors": "^1.0.2",
+    "react-transform-hmr": "^1.0.4",
+    "redbox-react": "^1.3.0",
+    "rimraf": "^2.5.4",
+    "sinon": "^1.17.5",
+    "sinon-chai": "^2.8.0",
+    "style-loader": "^0.13.1",
+    "superagent": "^2.2.0",
+    "webpack": "^1.13.2",
+    "webpack-dev-middleware": "^1.7.0",
+    "webpack-hot-middleware": "=2.12.2"
+  },
+  "config": {
+    "commitizen": {
```

Comment:
as you like, if it's a tool useful to you I don't have any objections adding it …